lib/class-wp-rest-block-editor-settings-controller.php

```diff
@@ -75,8 +75,8 @@ public function get_items_permissions_check( $request ) {// phpcs:ignore Variabl
 	 * @return WP_Error|WP_REST_Response Response object on success, or WP_Error object on failure.
 	 */
 	public function get_items( $request ) {
-		$context  = ! empty( $request['context'] ) ? $request['context'] : 'post-editor';
-		$settings = gutenberg_get_block_editor_settings( $context );
+		$editor_context  = new WP_Block_Editor_Context();
```

Comment:
@geriux, we are trying an approach with the context class. This way it should be simpler to set a very customized context even when using REST API. It might break the existing logic for the mobile app, but we should iterate on it as new screens get exposed to the users.

Reply:
Sounds good! I tested this PR and the functionality still works correctly 👍